users/models.py

```diff
@@ -1,4 +1,14 @@
 from django.contrib.auth.models import AbstractUser
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.authtoken.models import Token
+
 
 class User(AbstractUser):
```

Comment:
Instead of creating your own `User` you could use Django's built-in ;)

Reply:
Let me explain why I did it such way. I just remembered such recommendation from django docs: "If you’re starting a new project, it’s highly recommended to set up a custom user model, even if the default User model is sufficient for you. This model behaves identically to the default user model, but you’ll be able to customize it in the future if the need arises". 
Should I change it back to built-in user or leave it as it is?